tools/chocolateyinstall.ps1

```diff
@@ -9,12 +9,32 @@ $url             = 'https://dl.influxdata.com/telegraf/releases/telegraf-1.4.0_w
 $url64           = 'https://dl.influxdata.com/telegraf/releases/telegraf-1.4.0_windows_amd64.zip'
 $fileLocation    = Join-Path $install_folder 'telegraf.exe'
 
+# Make sure the config directory exists
 If(!(Test-Path -Path $configDirectory)){
   New-Item -Path $configDirectory -ItemType Directory
 }
 
-If (Get-Service -Name "telegraf" -ErrorAction SilentlyContinue) {
+# If telegraf.exe exists, and the service is running, stop the service
+If (Test-Path -Path $fileLocation){
```

Comment:
Can you combine the multiple conditionals into one since they're all doing the same checks?

Lines 18-19 and 26-27

Reply:
Sure - I'll get that done